src/shrub/v2/command.py

```diff
@@ -0,0 +1,663 @@
+"""Configuration for commands in shrub."""
```

Comment:
There is a lot of code in this file, but most of it is boilerplate to give the different types of evergreen commands python structure.